settings.yaml

```diff
@@ -0,0 +1,36 @@
+BlockchainSettingsService: 
+    Keys:
+        settings-key: BlockchainSettingsService-Keys
+        types: 
+            - JsonArray
+    Db: 
+        LogsConnectionString: 
+            settings-key: BlockchainSettingsService-Db-Logs
+            types: 
+                - AzureTableStorage
+        DataConnectionString:
+            settings-key: BlockchainSettingsService-Db-Data
+            types:
+                - AzureTableStorage
+    RedisCache: 
+        RedisConfiguration: 
+            settings-key: BlockchainSettingsService-Redis
+            types:
+                - Redis
+        InstanceName: 
```

Comment:
Make this field optional in setting model, please. It's necessary only if multiple environments should be deployed in the same network

Reply:
I meant, that exactly this field https://github.com/LykkeCity/Lykke.Service.BlockchainSettings/blob/3d7d4852d11b41c055f8ea3a9e0358420ef37ad5/src/Lykke.Service.BlockchainSettings.Shared/Settings/ServiceSettings/RedisSettings.cs#L5 should be optional.

Reply:
btw, there is file name and class name mismatch in RedisSettings.cs